### PR TITLE
Update name and publisher

### PR DESCRIPTION
**Pull request by lziq**
Issue #51 , if available:

Description of changes:

Changed publisher and extension name.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
 	"name": "ion-lsp-client",
 	"description": "VSCode LSP client for Ion. ",
-	"author": "ion-team",
+	"author": "TheIonTeam",
```
Comment (therapon) [inline client/package.json:4]: I think this can be `"The Ion Team" or "Ion Team"

File: client/package.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
 	"name": "ion-lsp-client",
```
Comment (therapon) [inline client/package.json:2]: lets make this `ion-extension`

File: client/package.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 	"description": "VSCode LSP client for Ion. ",
```
Comment (therapon) [inline client/package.json:3]: And make this `Ion Extension to support the Ion data format on VSCode`

File: package.json [hunk 4]
```diff
@@ -1,14 +1,14 @@
 {
-	"name": "ion-lsp",
-	"description": "LSP for ion",
-	"author": "ion-team",
+	"name": "ion-extension",
+	"description": "VSCode extension for ion",
+	"author": "TheIonTeam",
```
Comment (therapon) [inline package.json:4]: same comment here

File: client/package.json (hunk 3 above)
Comment (lziq) [reply client/package.json:3]: This description is for the client. I will change the description for the whole extension.

**Pull request merged**